modules/terraform/init_backend_file_test.go

```diff
@@ -0,0 +1,31 @@
+package terraform
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/files"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInitBackendConfigFile(t *testing.T) {
+	t.Parallel()
+
+	testFolder, err := files.CopyTerraformFolderToTemp("../../test/fixtures/terraform-backend-file", t.Name())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	customBackendFile := filepath.Join(testFolder, "terraform.backend")
+
+	options := &Options{
+		TerraformDir: testFolder,
+		BackendConfig: map[string]interface{}{
+			customBackendFile: nil,
+		},
+	}
+
+	InitAndApply(t, options)
+
+	assert.FileExists(t, customBackendFile)
```

Comment:
This is actually testing if `CopyTerraformFolderToTemp` works, instead of the functionality of the backend configuration file. Can you:

- Update the configuration file to specify a different path for the local terraform state
- Verify that the configuration was successfully overridden by checking for the existence of the custom terraform state file

Reply:
Yeaah... Okay, I got it. Yes, will do.